#! /bin/sh /usr/share/dpatch/dpatch-run
## 30_stolen-from-head_fix_Transform.dpatch by Henning Glawe <glaweh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix broken transform.pd (reported on pdl-porters by Xavier Calbet <xcalbet@yahoo.es>)

@DPATCH@
diff -urNad pdl-2.4.2/Lib/Transform/transform.pd /tmp/dpep.ypqaAs/pdl-2.4.2/Lib/Transform/transform.pd
--- pdl-2.4.2/Lib/Transform/transform.pd	2004-12-17 20:49:34.000000000 +0100
+++ /tmp/dpep.ypqaAs/pdl-2.4.2/Lib/Transform/transform.pd	2005-01-15 13:50:53.000000000 +0100
@@ -1507,7 +1507,7 @@
   ##############################
   ## Rubber meets the road: calculate the inverse transformed points.
   my $ndc = PDL::Basic::ndcoords(@dd);
-  my $idx = $me->invert($ndc->PDL_Double + 0.5);
+  my $idx = $me->invert($ndc->double + 0.5);
 
   barf "map: Transformation had no inverse\n" unless defined($idx);
 	
